google/oauth2/reauth.py

```diff
@@ -334,6 +334,10 @@ def refresh_grant(
     response_status_ok, response_data, retryable_error = _client._token_endpoint_request_no_throw(
         request, token_uri, body, headers=metrics_header
     )
+
+    if not response_status_ok and isinstance(response_data, str):
```

Comment:
enable_reauth_refresh flag is ignored here. is that fine? shouldn't enable_reauth_refresh flag be considered for whether exception needs to be raised or not?